app/models/user.py

```diff
@@ -57,10 +57,11 @@ class User(UserMixin, db.Model):
     def __init__(self, **kwargs):
         super(User, self).__init__(**kwargs)
         if self.role is None:
```

Comment:
Otherwise, though, the PR looks good. I can still understand why Ben thinks it might not be a necessary change, but it streamlines enough stuff that I'm all for it. I'm gotta go, but the code is clear and simple, this can definitely be merged.